docs/legacy/redirects.asciidoc

```diff
@@ -0,0 +1,14 @@
+["appendix",role="exclude",id="redirects"]

```

Comment:
This file needs to belong to a book and not the entire directory. I'd move this to `en/observability/redirects.asciidoc`

Reply:
You can then update line 156 in [docs/en/observability/index.asciidoc](https://github.com/elastic/observability-docs/pull/1677/files#diff-684c7d57d45e7d7c685a504568d48f8e3988685b66c4391800b7c9f6c029db9e) to `include::redirects.asciidoc[]`. That should fix the build error:

```
05:35:17 INFO:build_docs:asciidoctor: ERROR: index.asciidoc: line 156: include file not found: /tmp/docsbuild/IWsPMW5v7p/observability-docs/docs/en/observability/legacy/redirects.asciidoc
```

Reply:
Updated and builds ok!